float32 conf
float32 class_id
float32[4] bbox
